### PR TITLE
Docs: Migrate TextArea, Pog, Video, Image, Status, Layer examples to Sandpack

### PR DESCRIPTION
**Pull request by diyorbek**
### Summary

Migrated the examples of TextArea, Pog, Video, Image, Status, Layer components to Sandpack.

### Links

- [Jira](https://jira.pinadmin.com/browse/GESTALT-6271)
- [TDD](link to Paper doc)
- [Figma](link to Figma file)

### Checklist

- [n/a] Added unit and Flow Tests
- [n/a] Added documentation + accessibility tests
- [n/a] Verified accessibility: keyboard & screen reader interaction
- [n/a] Checked dark mode, responsiveness, and right-to-left support
- [n/a] Checked stakeholder feedback (e.g. Gestalt designers)

### CODE REVIEW

File: docs/pages/web/status.js [hunk 1]
```diff
@@ -153,25 +193,24 @@ If using \`title\` to describe what the icon represents, \`accessibilityLabel\`
...
-  <Status type='queued' title='Queued' />
-  <Status type='inProgress' title='In progress' />
-  <Status type='halted' title='Halted' />
-  <Status type='ok' title='OK' />
-  <Status type='canceled' title='Canceled' />
-  <Status type='warning' title='Warning' />
-  <Status type='problem' title='Problem' />
-</Flex>
-`}
+            sandpackExample={
```
Comment (AlbertCarreras) [inline docs/pages/web/status.js:196]: can we do this in 2 columns?

**Review by AlbertCarreras — APPROVED** [review]
Thanks! Just a couple minor comments

File: docs/examples/textarea/withRefExample.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+// @flow strict
```
[the remark this answers is not in the document]
Comment (diyorbek) [reply docs/examples/textarea/withRefExample.js:1]: Is there any specific reason to remove it?

[the remark this answers is not in the document]
Comment (AlbertCarreras) [reply docs/examples/textarea/withRefExample.js]: It's not a good example. We don't put popover on TextArea. We are starting to rething ref examples in the variant sections. They tend to be bad examples.

**Pull request merged**